bindings/cs/MongoDB.Libmongocrypt/Package.include.template.csproj

```diff
@@ -27,6 +27,12 @@
       <PackagePath>x64/native/windows/mongocrypt.dll</PackagePath>
       <Pack>true</Pack>
     </Content>
+
+    <Content Include="@mongocrypt_BINARY_DIR@/RelWithDebInfo/mongocrypt.dll" Condition="Exists('@mongocrypt_BINARY_DIR@/RelWithDebInfo/mongocrypt.dll')">
```

Comment:
This change is needed because for the build to succeed on Evergreen

Reply:
Should there also be a `Condition="'$(Configuration)'=='Release'"` and `Condition="'$(Configuration)'!='Release'"` one line 25 in case both dlls were built? With this change Evergreen will only be building RelWithDebInfo. But if we started building Debug in addition, perhaps this would choose the Debug build?

Reply:
Hm, nice future proofing idea Kevin! 😸 I've added two extra commits needed for this idea to work (prior to these commits the C# binding was building and testing in debug mode with the RelWithDebInfo `mongocrypt.dll`